example/Cargo.toml

```diff
@@ -6,4 +6,4 @@ authors = ["Alex Pikalov <alex.pikalov.khar@gmail.com>"]
 [dependencies]
 time = "0.1.36"
 cdrs_helpers_derive = { path = ".." }
-cdrs = "2.0.0-beta.3"
+cdrs = "2.3.2"
```

Comment:
Note: this can only be merged back when https://github.com/AlexPikalov/cdrs/pull/330 has been merged